apps/app.go

```diff
@@ -49,8 +58,20 @@ func (app *App) IsConsulApp() bool {
 	return ok
 }
 
-func (app *App) ConsulName() string {
-	if value, ok := app.Labels[MARATHON_CONSUL_LABEL]; ok && !isSpecialConsulNameValue(value) {
+func (app *App) HasSameConsulNamesAs(other *App) bool {
+	return reflect.DeepEqual(rawConsulNames(app), rawConsulNames(other))
```

Comment:
Got it! I'll prefer to replace reflect with simple loop and do not rely on `reflect` package but I don't insist.